webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5369,18 +5373,20 @@ var CURRENT_GROUP_DATA={};
             };
 
             _this.getImageUrlForGroupType = function (contact, displayName) {
-                return contact.imageUrl? '<img src="' + contact.imageUrl + '"/>' :  _this.getContactImageByAlphabet(displayName);
+                var profileDisplayName = displayName ? displayName + ' profile image' : 'Profile image';
```

Comment:
@CaptainT33mo will it work if the display name contains special characters or if it contains a single quote ' such as  D'Souza

Reply:
@devashishmamgain yes. Not sure how the different screen readers will read it out aloud in their accents but yes it will work.